content/en/case-studies/enhancing-power-systems.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Enhancing Power Systems"
```

Comment:
In Figma, the title lists `Enhancing`, so I added `Power Systems`. We can update this if another title would be preferred.